package.json

```diff
@@ -26,17 +28,23 @@
   },
   "homepage": "https://github.com/openedx/eslint-config#readme",
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "@typescript-eslint/parser": "^5.58.0",
     "eslint": "8.38.0",
     "eslint-config-airbnb": "19.0.4",
+    "eslint-config-airbnb-typescript": "^17.0.0",
     "eslint-plugin-import": "2.27.5",
     "eslint-plugin-jsx-a11y": "6.7.1",
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0"
   },
   "peerDependencies": {
-    "eslint": "^6.8.0 || ^7.0.0 || ^8.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "@typescript-eslint/parser": "^5.58.0",
+    "eslint": "^7.32.0 || ^8.2.0",
```

Comment:
BREAKING CHANGE: `eslint@^6.8.0` was removed from peer dependencies as its no longer supported with the introduction of `eslint-config-airbnb-typescript`.